test/wasm/array.js

```diff
@@ -0,0 +1,6 @@
+const blob = WScript.LoadBinaryFile('array.wasm');

```

Comment:
You should add this test to `rlexe.xml` you will need to add a baseline file for it too.